client/client_test.go

```diff
@@ -16,38 +16,27 @@ package pd
 
 import (
 	"context"
+	"reflect"
 	"testing"
 	"time"
 
-	. "github.com/pingcap/check"
 	"github.com/pingcap/errors"
 	"github.com/pingcap/kvproto/pkg/pdpb"
-	"github.com/tikv/pd/client/testutil"
-	"go.uber.org/goleak"
+	"github.com/stretchr/testify/require"
 	"google.golang.org/grpc"
 )
 
-func Test(t *testing.T) {
-	TestingT(t)
+func TestTsLessEqual(t *testing.T) {
+	re := require.New(t)
+	re.True(tsLessEqual(9, 9, 9, 9))
+	re.True(tsLessEqual(8, 9, 9, 8))
+	re.False(tsLessEqual(9, 8, 8, 9))
+	re.False(tsLessEqual(9, 8, 9, 6))
+	re.True(tsLessEqual(9, 6, 9, 8))
 }
 
-func TestMain(m *testing.M) {
```

Comment:
We don't need it anymore?

Reply:
Ops, it seems I deleted it accidentally.